wifiphisher/extensions/lure10.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, shared_data):
 
         :param self: A Lure10 object
         :param data: Shared data from main engine
-        :type self: Deauthentication
+        :type self: Lure10
```

Comment:
Is there a specific reason you made these changes here. For sake of clarity this should be removed from this PR.

Reply:
The main problem is whenever we make functional changes that are not described in a commit. If one of these irrelevant changes breaks the software we will have a hard trouble finding it. But this is only a typo change and I doubt it will introduce any issues.

Reply:
fair enough 😄.